python/GafferUITest/TestCase.py

```diff
@@ -169,6 +169,9 @@ def assertNodeUIsHaveExpectedLifetime( self, module ) :
 			weakScript = weakref.ref( script )
 
 			nodeGadget = GafferUI.NodeGadget.create( script["node"] )
+			if not nodeGadget :
+				continue
```

Comment:
This means we're skipping the tests for `weakNodeUI` just because a NodeGadget isn't registered. Could you move the `assertIsNone` for that above here so it still runs?

Reply:
Yep, that's great - thanks Ivan!